app/models/user.rb

```diff
@@ -121,7 +124,8 @@ def find_permitted_users(filters = nil, pagination = nil, sort = nil, user = nil
       if filters.present?
         filters = filters.compact
         filters['disabled'] = filters['disabled'].values if filters['disabled'].present?
-        users = users.where(filters)
+        users = User.by_user_group_unique_ids(filters['user_group_ids']) if filters.has_key?('user_group_ids')
+        users = users.where(filters.except('user_group_ids'))
```

Comment:
Change those 2 lines to use the method suggested above:

```ruby
users = users.where(filters.except('user_group_ids'))
users = filter_with_groups(users, filters)
```

Reply:
Okay, Done